IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDClientTLSHandler.m

```diff
@@ -44,6 +44,10 @@ + (BOOL)handleChallenge:(NSURLAuthenticationChallenge *)challenge
                 context:(id<MSIDRequestContext>)context
       completionHandler:(ChallengeCompletionHandler)completionHandler
 {
+
+#if TARGET_OS_IPHONE
```

Comment:
let's see if we can disable entire MSIDTLSHandler for iOS with the same result, as discussed offline

Reply:
Since the MSIDClientTLSHandler's handleChallenge API is called by MSIDChallengeHandler's handleChallenge as below, and since the handler may not always be TLS handler, I think it would be safer and much easier to have TLShandler's handle challenge return NO when it's on iOS, instead of disabling it altogether in below calling API.

   
    BOOL handled = NO;
    Class<MSIDChallengeHandling> handler = nil;
    @synchronized (self)
    {
        handler = [s_handlers objectForKey:authMethod];
    }
    
    if (!handler)
    {
        completionHandler(NSURLSessionAuthChallengePerformDefaultHandling, nil);
        return;
    }
    
    handled = [handler handleChallenge:challenge
                               webview:webview
                               context:context
                     completionHandler:completionHandler];